src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/Transformer.kt

```diff
@@ -165,17 +165,21 @@ fun CourtEventChargeResponse.toDpsMigrationCharge(
     chargeNOMISId = chargeId,
     sentence = dpsSentence,
     merged = if (isSourceCase) {
+      // TODO: this is not strickly correct - since this implies all court event charges on source are merged
+      // which may not be the case - however given I don't think DPS needs this revisit when
+      // requirements are established, it would probably require NOMIS API changes to support this correctly
       true
-    } else if (linkedCourtEvent != null) {
```

Comment:
Use the new linkedCaseDetails element read by NOMIS API that links the linked charge data